seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/sink/HbaseSinkFactory.java

```diff
@@ -59,4 +67,16 @@ public OptionRule optionRule() {
                         HBASE_EXTRA_CONFIG)
                 .build();
     }
+
+    @Override
+    public TableSink createSink(TableSinkFactoryContext context) {
+        TableSinkFactoryContext.replacePlaceholderAndCreate(
+                context.getCatalogTable(),
+                context.getOptions(),
+                Thread.currentThread().getContextClassLoader(),
+                Collections.emptyList());
```

Comment:
We already invoked this method in https://github.com/apache/seatunnel/blob/df210ea73d5b14b36b9637fc4d12bee09fa8decd/seatunnel-api/src/main/java/org/apache/seatunnel/api/table/factory/TableSinkFactoryContext.java#L51
So we shouldn't invoke it in connector. Please remove it.